core/types/client/opts.go

```diff
@@ -24,6 +24,10 @@ type Options struct {
 	// communication with a trusted node (using TLS or Unix sockets).
 	ChainID string
 
+	// TrustLocalChainID will trust the chain ID passed from the user, i.e. no check
+	// against remote node's chain ID. This option is only be effective when ChainID is set.
+	TrustLocalChainID bool
```

Comment:
I'm not really like this name, any suggestion?

Reply:
`SkipVerifyChainID` maybe